tests/e2e/controlplane/control_plane_test.go

```diff
@@ -355,3 +392,47 @@ func forceDeleteIstioResources() error {
 
 	return nil
 }
+
+func getBookinfoYAML(version supportedversion.VersionInfo) (string, error) {
```

Comment:
Since our `kubectl` is just a thin wrapper around the real `kubectl` command, can't we just pass the url directly, as in `kubectl apply -f https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/platform/kube/bookinfo.yaml` ? That way we don't need to mess with http reponses and such